core/modules/widgets/fieldmangler.js

```diff
@@ -135,6 +135,10 @@ FieldManglerWidget.prototype.handleAddTagEvent = function(event) {
 			$tw.utils.pushTop(modification.tags,tag);
 			this.wiki.addTiddler(new $tw.Tiddler(tiddler,modification));			
 		}
+	} else if(typeof event.param === "string" && event.param.trim() !== "" && this.mangleTitle.trim() !== "") {
```

Comment:
There is / was a discussion about trimming tiddler titles. ... So we may introduces unwanted side effects here. 

see my issue:  Tiddler title handling should trim whitespace #751  .... The "conclusion" was "white space in tiddler titles is valid"

So imo the core needs to handle those clashes!